rita_exit/src/database/mod.rs

```diff
@@ -402,6 +451,51 @@ pub fn setup_clients(
     Ok(wg_clients)
 }
 
+fn delete_generic_ipv4_route() {
+    // Delete generic ipv4 routes (Remove hardcoding)
+    KI.run_command("ip", &["route", "del", "172.168.0.0/16", "dev", "wg_exit"])
+        .expect("Fix command");
+    KI.run_command(
+        "ip",
+        &["route", "del", "172.168.0.0/16", "dev", "wg_exit_new"],
+    )
+    .expect("Fix command");
+}
```

Comment:
I couldnt find where these routes were being set in the first place. If you point me to those, I can remove this code